package.json

```diff
@@ -34,6 +34,7 @@
         "immutability-helper": "^2.9.0",
         "jest": "^23.6.0",
         "lodash": "^4.17.11",
+        "material-ui": "^0.20.2",
```

Comment:
They are actually using 0.17. See https://github.com/dhis2/maintenance-app/blob/v30/package.json#L83 Not sure if it matters... guys?

Reply:
This is not related to HeaderBar, we have material-ui for the legacy theme only but the dependency was not tracked. There were no issues because we loaded as peer dependency of another package but anyway we should track it.

Reply:
Capito! Thanks!